static/app/views/alerts/issueRuleEditor/ruleNode.tsx

```diff
@@ -393,7 +401,16 @@ class RuleNode extends React.Component<Props> {
                 icon={<IconSettings size="xs" />}
                 type="button"
                 onClick={() => {
-                  // TODO(nisanthan): Placeholder. Modal will be implemented in next PR.
+                  openModal(deps => (
+                    <SentryAppRuleModal
+                      {...deps}
+                      sentryAppInstallationUuid={node.sentryAppInstallationUuid || ''}
```

Comment:
@leeandher if this is a sentryapp, it should always have the uuid.

Reply:
thanks, fixing this